src/anomalib/utils/visualization/image.py

```diff
@@ -274,6 +286,22 @@ def _visualize_simple(self, image_result: ImageResult) -> np.ndarray:
             else:
                 image_classified = add_normal_label(image_result.image, 1 - image_result.pred_score)
             return image_classified
+
+        if self.task == TaskType.EXPLANATION:
+            image_grid = _ImageGrid()
+            description = ""
+            if image_result.text_descr:
+                description = image_result.text_descr
+
+            if image_result.pred_label:
```

Comment:
I am a bit confused about this. I know this is similar to the classification task. But `pred_label` is of type `str`. How does this exactly work?

Reply:
Actually, I think that pred_label is a boolean from:

src/anomalib/callbacks/post_processor.py : 80
```python
if "pred_scores" in outputs:
    outputs["pred_labels"] = outputs["pred_scores"] >= pl_module.image_threshold.value```
Maybe is bad typed in the ImageResult class.